site/en/docs/extensions/mv3/getstarted/development-basics/index.md

```diff
@@ -0,0 +1,269 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Development basics'
+subhead: 'Learn what the development of a Chrome extension will look like.'
+description: 'What to expect during the development of a Chrome extension.'
+date: 2022-08-10
+# updated: 2022-06-13
+---
+
+## Overview {: #overview }
+
+This page describes the extension development workflow. We will create a "Hello, Extensions"
+example, load the extension locally, locate logs and errors and share other development
+recommendations.
+
+## Hello Extensions {: #build }
+
+This extension will display “Hello Extensions” when the user clicks on the extension toolbar icon.
+
+<figure>
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/xjKRmWMgwMm7Kdf72bkj.png", 
+alt="Hello extension", width="206", height="130", class="screenshot" %}
+  <figcaption>
+  Hello Extension popup
+  </figcaption>
+</figure>
+
+Start by creating a new directory to store extension files. If you prefer, you can download the full
+source code hosted on [Github][sample-hello-world].
+
+Next, create a new file in this directory called `manifest.json` and add the following code:
+
+{% Label %}manifest.js:{% endLabel %}
+
+```json
+{
+  "manifest_version": 3,
+  "name": "Hello Extensions",
+  "description": "Base Level Extension",
+  "version": "1.0",
+  "action": {
+    "default_popup": "hello.html",
+    "default_icon": "hello_extensions.png"
+  }
+}
+```
+
+This JSON object describes the extension's capabilities and configuration. For example, the `"action"` key 
+declares the image Chrome should use as the action's icon and the HTML page to show in a popup when the action is clicked. You can download the icon [here][hello-icon]. 
+
+For the popup, create a file named `hello.html`, and add the following code:
+
+{% Label %}hello.html{% endLabel %}
+
+``` html
+<html>
+  <body>
+    <h1>Hello Extensions</h1>
+  </body>
+</html>
+```
+
+The extension now displays a popup when the extension action (toolbar icon) is clicked. Let's test
+it on Chrome by loading it locally. Ensure all files are saved.
+
+## Loading an unpacked extension {: #load-unpacked }
+
+To load an unpacked extension in developer mode, follow these steps:
+
+1. Go to the Extension Management page by navigating to `chrome://extensions`.
```

Comment:
I use this term because I found it in the following documentation pages:
- [manifest - description](https://developer.chrome.com/docs/extensions/mv3/manifest/description/#:~:text=browser%27s%20extension%20management)
- [Give users options](https://developer.chrome.com/docs/extensions/mv3/options/#:~:text=extension%20management%20page)
- [manifest - name](https://developer.chrome.com/docs/extensions/mv2/manifest/name/#:~:text=Extension%20management%20UI)

Looking at the UI, the page does name it simply "Extensions", so I see where you are coming from.

<img width="800" alt="Screen Shot 2022-09-09 at 11 06 36" src="https://user-images.githubusercontent.com/37001393/190209062-67d9f762-0780-435e-b4ac-e9ad42fa49c0.png">

My only concern is that we call "extension pages" to refer to HTML pages included in the extension package.

Before making all these changes, I'd like to consult the team to ensure we all agree on this term change and perhaps start building a Chrome Extension Glossary that will eventually live on the docs.

Reply:
Does extension pages refer generically to all pages used in an extension? In other words, is it a super set of content and popup pages?

Reply:
In the [Extension Architecture](https://developer.chrome.com/docs/extensions/mv3/architecture-overview/#files), we use extension pages to refer to any HTML page included in the extension (index.html, popup.html, onboarding.html, etc).

I found this [Google support on managing extensions](https://support.google.com/chrome_webstore/answer/2664769?hl=en#zippy=%2Cinstall-on-your-phone%2Cinstall-with-enhanced-safe-browsing), and it looks like you might be right about calling it simply "Extensions" page.

I think you won me over on this one. I'll make the changes.

Reply:
Let's leave it for now. Giving something a different name in documentation than it has in the interface is not a best practice. Long term, we need to find another way to disambiguate these two items.